lib/adviser_list_csv.rb

```diff
@@ -5,11 +5,33 @@ def initialize(advisers)
 
   def to_csv(_ = {})
     CSV.generate do |csv|
-      csv << ['Ref. Number', 'Name', 'Firm']
-
+      csv << headings
       @advisers.each do |adviser|
-        csv << [adviser.reference_number, adviser.name, adviser.firm.registered_name]
+        csv << row(adviser)
       end
     end
   end
+
+  private
+
+  def headings
+    ['Ref. Number', 'Name', 'Firm'] + all_certifications
+  end
+
+  def all_certifications
+    @all_certifications ||= Accreditation.all.pluck(:name) + Qualification.all.pluck(:name)
+  end
+
+  def certified(adviser)
+    adviser_certifications = adviser.accreditations.pluck(:name) + adviser.qualifications.pluck(:name)
+    all_certifications.map { |name| adviser_certifications.include?(name) ? 'Y' : 'N' }
```

Comment:
Another minor. Can we change the map variable from `name` to `certification`? Wdyt?

Reply:
👍  sounds good, will change now